### PR TITLE
Edit/devdoc 221 rename backoffice

### PR DESCRIPTION
**Pull request by graz-dev**
## Description

Changes all the occurrences of "backoffice" in "microfrontend composer"

## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensitive content has been committed

### CODE REVIEW

File: docs/console/project-configuration/create-a-project.mdx [hunk 1]
```diff
@@ -149,7 +149,7 @@ The Project creation process is divided into three phases:
    * **Environment name**: the name given to your environment;
    * **Environment ID**: the human-readable ID set to your environment;
    * **Project Host**: the host which exposes documentation and APIs;
-   * **Backoffice Host**: the host which exposes CMS and backoffice APIs;
+   * **Microfrontend Composer Host**: the host which exposes CMS and Microfrontend Composer APIs;
```
Comment (akelity) [inline docs/console/project-configuration/create-a-project.mdx:152]: Are you sure they changed the label of this variable and that it's now called this?

File: docs/getting-started/mia-platform-overview.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Mia-Platform products are built **by developers for developers**, and you can [a
 - Monitor runtime resources usage and optimize them;
 - Expose all the APIs through an API Gateway of your choice, in an easy and secure way;
 - Leverage a Data Fabric solution to govern the entire **data lifecycle**. Project data from your legacy systems and create single views - i.e. unified views of a subset of data with a specific purpose - to reduce access latency, and increase data availability;
-- Create and evolve an entire **backoffice application** in a few clicks using the low-code features to read and write your data.
+- Create and evolve an entire **microfrontend composer application** in a few clicks using the low-code features to read and write your data.
```
Comment (akelity) [inline docs/getting-started/mia-platform-overview.md:31]:
```suggestion
- Create and evolve an entire **Microfrontend Composer application** like an backoffice application (and much more) in a few clicks using the low-code features to read and write your data.
```

File: docs/getting-started/tutorials/index.md [hunk 3]
```diff
@@ -31,10 +31,10 @@ Welcome to the Tutorials page! This page contains a series of tutorials guides d
 
 
 
-### **Backoffice**
-- **Backoffice Basics**: Understanding the basic functionalities of the backoffice. [Go to Backoffice Basics](/getting-started/tutorials/microfrontend-composer/basics.mdx).
-- **Backoffice Templates**: Learning to use templates in the backoffice. [Go to Backoffice Templates](/getting-started/tutorials/microfrontend-composer/templates.mdx).
-- **Backoffice Microfrontends**: Delving into microfrontends in the backoffice. [Go to Backoffice Microfrontends](/getting-started/tutorials/microfrontend-composer/microfrontends.mdx).
+### **Microfrontend Composer**
+- **Microfrontend Composer Basics**: Understanding the basic functionalities of the microfrontend composer. [Go to Microfrontend Composer Basics](/getting-started/tutorials/microfrontend-composer/basics.mdx).
```
Comment (akelity) [inline docs/getting-started/tutorials/index.md:35]:
```suggestion
- **Microfrontend Composer Basics**: Understanding the basic functionalities of the Microfrontend Composer. [Go to Microfrontend Composer Basics](/getting-started/tutorials/microfrontend-composer/basics.mdx).
```

File: docs/getting-started/tutorials/index.md [hunk 4]
```diff
@@ -31,10 +31,10 @@ Welcome to the Tutorials page! This page contains a series of tutorials guides d
...
+- **Microfrontend Composer Templates**: Learning to use templates in the microfrontend composer. [Go to Microfrontend Composer Templates](/getting-started/tutorials/microfrontend-composer/templates.mdx).
```
Comment (akelity) [inline docs/getting-started/tutorials/index.md:36]:
```suggestion
- **Microfrontend Composer Templates**: Learning to use templates in the Microfrontend Composer. [Go to Microfrontend Composer Templates](/getting-started/tutorials/microfrontend-composer/templates.mdx).
```

File: docs/getting-started/tutorials/index.md [hunk 5]
```diff
@@ -31,10 +31,10 @@ Welcome to the Tutorials page! This page contains a series of tutorials guides d
...
+- **Microfrontend Composer Microfrontends Management**: Delving into microfrontends in the microfrontend composer. [Go to Microfrontend Composer Microfrontends](/getting-started/tutorials/microfrontend-composer/microfrontends.mdx).
```
Comment (akelity) [inline docs/getting-started/tutorials/index.md:37]: Microfrontend Composer Microfrontends ... Maybe we need to find a best way to say that...

File: docs/getting-started/videos/index.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ Welcome to the video page! This page contains a series of video guides designed
 * **Connecting Cluster**: A step-by-step guide to connect your cluster. [Go to connecting-cluster.mdx](/getting-started/videos/connecting-cluster.mdx).
 * **Feature Preview**: An overview of the upcoming features. [Go to feature-preview.mdx](/getting-started/videos/feature-preview.mdx).
 * **M2M Auth**: Everything you need to know about M2M authentication. [Go to m2m-auth.mdx](/getting-started/videos/m2m-auth.mdx).
-* **Backoffice**: How to navigate and effectively use the backoffice interface. [Go to backoffice.mdx](/getting-started/videos/backoffice.mdx).
+* **Microfrontend Composer**: How to navigate and effectively use the microfrontend composer interface. [Go to microfrontend-composer.mdx](/getting-started/videos/microfrontend-composer.mdx).
```
Comment (akelity) [inline docs/getting-started/videos/index.md:17]:
```suggestion
* **Microfrontend Composer**: How to navigate and effectively use the Microfrontend Composer interface. [Go to microfrontend-composer.mdx](/getting-started/videos/microfrontend-composer.mdx).
```

File: docs/getting-started/videos/microfrontend-composer.mdx [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
-id: backoffice
```
Comment (akelity) [inline docs/getting-started/videos/microfrontend-composer.mdx:2]: You need to write the redirect about that.
You will add the redirect on the PR: #793

File: docs/microfrontend-composer/overview.md [hunk 8]
```diff
@@ -42,9 +42,9 @@ Perhaps you need a CMS, one that’s malleable to your every whim and requiremen
 
 Whether you're in the business of constructing internal tools, unique dashboards, or intricate UIs, Mia-Platform Microfrontend Composer is your canvas, and your imagination is the limit.
 
-### Backoffice Applications
+### Microfrontend Composer Applications
 
-In many businesses, backoffice applications play a pivotal role in ensuring smooth operations. These applications handle the internal processes and tasks that aren't seen by the customers but are vital for the organization's functioning. Mia-Platform Microfrontend Composer offers an optimal environment to create and manage these critical applications with precision and ease.
+In many businesses, microfrontend composer applications play a pivotal role in ensuring smooth operations. These applications handle the internal processes and tasks that aren't seen by the customers but are vital for the organization's functioning. Mia-Platform Microfrontend Composer offers an optimal environment to create and manage these critical applications with precision and ease.
```
Comment (akelity) [inline docs/microfrontend-composer/overview.md:47]:
```suggestion
In many businesses, Microfrontend Composer applications play a pivotal role in ensuring smooth operations. These applications handle the internal processes and tasks that aren't seen by the customers but are vital for the organization's functioning. Mia-Platform Microfrontend Composer offers an optimal environment to create and manage these critical applications with precision and ease.
```

**Review by akelity — CHANGES_REQUESTED** [review]
TBF

File: docs/console/project-configuration/create-a-project.mdx (hunk 1 above)
Comment (graz-dev) [reply docs/console/project-configuration/create-a-project.mdx:152]: You are right, it was correctly written as "Backoffice"

**Review by akelity — APPROVED** [review]
LGTM

**Pull request merged**